packages/docusaurus-plugin-content-pages/src/index.ts

```diff
@@ -209,7 +206,10 @@ export default function pluginContentPages(
                     rehypePlugins,
                     beforeDefaultRehypePlugins,
                     beforeDefaultRemarkPlugins,
-                    staticDir: path.join(siteDir, STATIC_DIR_NAME),
+                    staticDirs: siteConfig.staticDirectories.map((dir) =>
+                      path.join(siteDir, dir),
```

Comment:
Wonder if we shouldn't use `path.resolve` instead, so that we could support `staticDirectories` with absolute paths too?

![image](https://user-images.githubusercontent.com/749374/142403697-f9fa239a-9a97-4baf-ac01-e501240ae9fd.png)